src/client-lib/utils/rendezvous-socket.js

```diff
@@ -0,0 +1,43 @@
+import EventEmitter from 'eventemitter3';
+
+class RendezvousSocket extends EventEmitter {
```

Comment:
I tried this in the past. I really want to be able to do `rendezvousSocket.on('msg')` but wasn't able to do so unless I add inheritance. I don't want to make the user do `rendezvousSocket.events.on('msg')`